src/software/ai/evaluation/robot.h

```diff
@@ -68,3 +68,5 @@ std::optional<bool> robotBeingPassedTo(const BallHistory& ball_states,
 template <typename STATE_TYPE>
 std::optional<STATE_TYPE> findState(boost::circular_buffer<STATE_TYPE> states,
                                     Timestamp& timestamp);
+
+bool pointNearRobot(Point point, Robot robot);
```

Comment:
function should have javadoc

Also I find the name misleading compared to what the function seems to be doing. Based on the name I assume it checks if the given point is within some distance of the robot. However the function is actually checking for possession. Would be good to have a more descriptive name like `ballNearRobotMouth`, or `robotHasPossessionOfBall` if it's supposed to be for possession.

Reply:
this was originally pointNearDribbler in the other PR, would that be better?

Reply:
This function is identical to the `isNearDribbler` function in #1768, depending on which goes PR goes in first we should probably use one or the other

Reply:
yea pointNearDribbler would also be better